components/director/internal/domain/onetimetoken/resolver_test.go

```diff
@@ -185,3 +187,106 @@ func TestResolver_GenerateOneTimeTokenForRuntime(t *testing.T) {
 		conv.AssertExpectations(t)
 	})
 }
+
+func TestResolver_RawEncoded(t *testing.T) {
+	testErr := errors.New("test error")
+	txGen := txtest.NewTransactionContextGenerator(testErr)
+	ctx := context.TODO()
+	tokenGraphql := graphql.OneTimeToken{Token: "Token", ConnectorURL: "connectorURL"}
+	expectedRawToken, err := json.Marshal(graphql.OneTimeToken{Token: "Token", ConnectorURL: "connectorURL"})
```

Comment:
hmm, maybe write this json manually, without marshalling it, because it's almost exactly the same as the function we're testing 😄 

Then encode it with base64 package.